app/assets/javascripts/controllers/container_dashboard/container_dashboard_controller.js

```diff
@@ -61,7 +61,8 @@ miqHttpInject(angular.module('containerDashboard', ['ui.bootstrap', 'patternfly'
           id = '';
         }
         else {
-          id = '/'+ (/\/(\d+)/.exec($location.absUrl())[1]);
+          // search for http://<url>/<controller>/<number>...
+          id = '/' + (/\/.+\/.+\/(\d+)/.exec($location.absUrl())[1]);
```

Comment:
This is not quite right.. too eager and unanchored.. for example..

`http://whatever/controller/1234?param=456` will return `456`, so will `http://whatever/controller/1234/456`.

If we want to *force* the url to look as `/<controller>/<number>`, why not use `$location.path()` and anchor the regex as `/^\/[^\/]+\/(\d+)$/`?